chain/scripts/start_yoda.sh

```diff
@@ -11,6 +11,9 @@ yoda config validator $(bandcli keys show $1 -a --bech val --keyring-backend tes
 # setup execution endpoint
 yoda config executor "docker:bandprotocol/runtime:1.0.1"
 
+# setup broadcast-timeout  to yoda config
+yoda config broadcast-timeout "10s"
```

Comment:
@sorawit 10 seconds is the default of BroadcastTxCommit if we want to solve this problem how much time is a recommended option?